src/pages/settings/Preferences/PriorityModePage.js

```diff
@@ -27,6 +27,7 @@ const PriorityModePage = (props) => {
                 value: key,
                 text: mode.label,
                 alternateText: mode.description,
+                alternateTextMaxLines: 0,
```

Comment:
@bernhardoj Can you please add some comments to explain this line.

Reply:
added

Reply:
Sorry, why would we ever want `alternateTextMaxLines` to be `0`? Wouldn't the minimum be `1`?

Reply:
We can set it to 0 to not restrict the maximum number of lines, so it will take as many lines as needed.
https://reactnative.dev/docs/text#numberoflines